cms/envs/common.py

```diff
@@ -712,6 +712,7 @@
     'BUCKET': '',
     'ROOT_PATH': '',
     'CONCURRENT_UPLOAD_LIMIT': 4,
+    'COURSE_VIDEO_UPLOAD_TOKEN': '',
```

Comment:
Could you make this `None`? I think it aligns more with the other `is not None` checks.